src/main/scala/org/broadinstitute/dsde/firecloud/dataaccess/HttpSamDAO.scala

```diff
@@ -37,4 +43,58 @@ class HttpSamDAO( implicit val system: ActorSystem, implicit val executionContex
     }
   }
 
+  override def createGroup(groupName: WorkbenchGroupName)(implicit userInfo: WithAccessToken): Future[Unit] = {
+    userAuthedRequest(Post(samManagedGroup(groupName))).map { resp =>
+      if(resp.status.isSuccess) ()
+      else throw new FireCloudExceptionWithErrorReport(FCErrorReport(resp))
+    }
+  }
+
+  override def deleteGroup(groupName: WorkbenchGroupName)(implicit userInfo: WithAccessToken): Future[Unit] = {
+    userAuthedRequest(Delete(samManagedGroup(groupName))).map { resp =>
+      if(resp.status.isSuccess) ()
+      else throw new FireCloudExceptionWithErrorReport(FCErrorReport(resp))
+    }
+  }
+
+  override def listGroups(implicit userInfo: WithAccessToken): Future[List[FireCloudManagedGroupMembership]] = {
+    authedRequestToObject[List[SamManagedGroupMembership]](Get(samManagedGroupsBase)).map { membership =>
+      membership.map(m => FireCloudManagedGroupMembership(m.resourceId, m.policyEmail, m.accessPolicyName.stripSuffix("s").capitalize))
```

Comment:
not ideal, but it keeps the new response identical to the old one

Reply:
Should SAM be modified to return a different result?

Reply:
that would be ideal

Reply:
Is the desired response described anywhere?  If you can make a JIRA ticket or tell me what you want, I'll make the ticket and make the necessary change(s).

Reply:
what are you hoping to change? Maybe change the policies to not be plural?

Reply:
the only change needed is for the policies to be called "admin" and "member" instead of "admins" and "members", and all of the related language in sam to be singular too